jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/PluginConfigurationProcessor.java

```diff
@@ -458,22 +454,37 @@ static ContainerizingMode getContainerizingModeChecked(
   static BiFunction<Path, AbsoluteUnixPath, Instant> createModificationTimeProvider(
       String modificationTime) throws InvalidFilesModificationTimeException {
     try {
-      switch (modificationTime) {
-        case "EPOCH_PLUS_SECOND":
-          Instant epochPlusSecond = Instant.ofEpochSecond(1);
-          return (ignored1, ignored2) -> epochPlusSecond;
-
-        default:
-          Instant timestamp =
-              DateTimeFormatter.ISO_DATE_TIME.parse(modificationTime, Instant::from);
-          return (ignored1, ignored2) -> timestamp;
-      }
+      Instant instant = getConfiguredTime(modificationTime);
```

Comment:
This should be the right, but just letting you know that originally we had `KEEP_ORIGINAL` for files which would return different timestamps determined in a later stage when files are actually added to a layer as `LayerEntry`. If we revive `KEEP_ORIGINAL`, then we will probably need to revert this back, but for now, I agree we should go for this code reuse. No action needed.